packages/elements/package.json

```
@@ -5,6 +5,7 @@
   "license": "Apache-2.0",
   "main": "lib/index.js",
   "module": "es/index.js",
+  "sass": "index.scss",
```

Should the entrypoints contain the current directory? The examples from the docs have it. 

```suggestion
  "sass": "./index.scss",
```

---

My read on it is that it's similar to Node's rules for resolving `main` and `module`, so I followed the same style that was already used for those. 

In the actual [Dart code](https://github.com/sass/dart-sass/blob/52221c0e1587c745d073eac961bc665dcbb6075f/lib/src/importer/node_package.dart#L147) it uses `path.join()` using the directory containing the `package.json` and the path part specified by `sass` entry, so I think this is safe.

---

Awesome, thanks!